src/STLViewPort.js

```diff
@@ -385,10 +385,23 @@ export function STLViewPort( canvas, width, height ) {
 
     self.recalculateOverhang = function(model) {
         if (!model || !model.orientationOptimizer) return;
+        if (model.userData.recalculateOverhangPreviousRotation) {
+            const DEBUGGING = false;
```

Comment:
I just put it there for my debugging.  When you turn it on, it displays on the console whether the overhang was recalculated or not.  I used it to check that the overhand is recalculated only when I expected.  I left it in there so that you could test it out and see the performance increase.

If you don't need to test it and want it gone, I can remove it.